README.md

```diff
@@ -68,6 +68,17 @@ For non-NixOS systems, install directly with:
 nix-env -iA nixpkgs.tenere
 ```
 
+### 📱 Mobile (nix-on-droid)
+
+Tenere works on Android via nix-on-droid ([demo](assets/tenere-on-droid.gif)).
```

Comment:
Can you upload this gif instead of having it assets directory ? That way we keep it out of git history  
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#uploading-assets

Reply:
thanks for teaching me that little trick just now